ops/services/02-config/values/prod-sbx.sops.yaml

```
@@ -83,25 +87,29 @@
 /bfd/${env}/ccw-pipeline/nonsensitive/ccw/new_relic/app_name: BFD Pipeline ccw (${env})
 /bfd/${env}/ccw-pipeline/nonsensitive/ccw/slo/weekend_data_availability/verifier/enabled: true
 /bfd/${env}/ccw-pipeline/nonsensitive/ccw/slo/weekend_data_availability/verifier/alert_topics: bfd-prod-sbx-cloudwatch-alarms-slack-bfd-warnings
-/bfd/${env}/pipeline/nonsensitive/rda/grpc/max_idle_seconds: "600"
-/bfd/${env}/pipeline/nonsensitive/rda/grpc/server_type: InProcess
-/bfd/${env}/pipeline/nonsensitive/rda/grpc/inprocess_server/mode: S3
-/bfd/${env}/pipeline/nonsensitive/rda/job/enabled: "true"
-/bfd/${env}/pipeline/nonsensitive/rda/job/interval_seconds: "3600"
-/bfd/${env}/pipeline/nonsensitive/rda/job/batch_size: "20"
-/bfd/${env}/pipeline/nonsensitive/rda/job/write_thread_count: "5"
-/bfd/${env}/pipeline/nonsensitive/rda/job/process_dlq: "true"
-/bfd/${env}/pipeline/nonsensitive/rda/samhsa/backfill/enabled: "false"
-/bfd/${env}/pipeline/nonsensitive/rda/samhsa/backfill/batch_size: 100000
-/bfd/${env}/pipeline/nonsensitive/rda/samhsa/backfill/log_interval: 900
-/bfd/${env}/pipeline/nonsensitive/rda/job/starting_fiss_seq_num: UNDEFINED
-/bfd/${env}/pipeline/nonsensitive/rda/job/starting_mcs_seq_num: UNDEFINED
-/bfd/${env}/pipeline/nonsensitive/rda/job/sequence_range_update_interval_seconds: 300
-/bfd/${env}/pipeline/nonsensitive/rda/new_relic/app_name: BFD Pipeline (${env})
-/bfd/${env}/pipeline/nonsensitive/rda/cleanup/enabled: "false"
-/bfd/${env}/pipeline/nonsensitive/rda/cleanup/run_size: UNDEFINED
-/bfd/${env}/pipeline/nonsensitive/rda/cleanup/transaction_size: UNDEFINED
-/bfd/${env}/pipeline/nonsensitive/rda/instance_type: m6a.large
+/bfd/${env}/rda-pipeline/nonsensitive/ecs/resources/cpu: 4096
+/bfd/${env}/rda-pipeline/nonsensitive/ecs/resources/memory: 8192
+/bfd/${env}/rda-pipeline/nonsensitive/ecs/resources/disk_size: 21
+/bfd/${env}/rda-pipeline/nonsensitive/micrometer_cw/enabled: "false"
+/bfd/${env}/rda-pipeline/nonsensitive/micrometer_cw/namespace: bfd-prod-sbx/rda-pipeline
```

Should these be parameterized?
```suggestion
/bfd/${env}/rda-pipeline/nonsensitive/micrometer_cw/namespace: bfd-${env}/rda-pipeline
```

---

Yep. I'll fix this up.